Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby File.read('.ruby-version').strip
+ruby file: ".ruby-version"
```

Comment:
I had a misleading error message regarding this line so I tried to change it.
However, it was not related.
Still we could keep the change in my opinion.
https://andycroll.com/ruby/read-ruby-version-in-your-gemfile/